VogtObserver2/Trader.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace VogtObserver2
+{
+    class Trader : IObserver
+    {
+        private string _name;
+
+        private Random _rand = new Random();
+
+        private string[] _buyOrSell = new string[] { " ", "buy", "sell" };
+
+        public Trader(string name)
+        {
+
+            _name = name;
+
+        }
+
+        public void Update(double cost, string symbol)
+        {
+            Console.WriteLine($"The Lastest trade is Trader: {_name} {_buyOrSell[_rand.Next(1, 3)]} cost {cost:C2} Stock: {symbol}");
```

Comment:
Ahhh I see. So youre acting like the trader randomly bought, sold, or did nothing with a particular symbol?

So all traders are watching all stocks and randomly choosing their action for the stock.